src/wp-admin/includes/class-wp-site-health.php

```diff
@@ -2359,6 +2477,12 @@ public static function get_tests() {
 					'has_rest'          => true,
 					'async_direct_test' => array( WP_Site_Health::get_instance(), 'get_test_https_status' ),
 				),
+				'page_cache'           => array(
```

Comment:
Maybe we should only add these checks on production?

Reply:
Should we limit it only to prod? what do you think @swissspidy @felixarntz

Reply:
@manuelRod I think limiting this check to production sites makes sense to me, as it's also about caching, which shouldn't really be an issue on development sites. I would also say it makes sense to do it in accordance with #2890 which does it only for production too.

Reply:
makes sense